web/locales/en/plugin__network-observability-plugin.json

```diff
@@ -8,6 +8,9 @@
   "Cancel": "Cancel",
   "At least one column must be selected": "At least one column must be selected",
   "Save": "Save",
+  "Compact": "Compact",
```

Comment:
oh, you already implemented this, cool! :-)

Reply:
yes it was a quick win :1st_place_medal: